packages/memoize/types/index.d.ts

```diff
@@ -1,3 +1,3 @@
-type Fn<T> = (key: string) => T
+export type Fn<T> = (key: string) => T
```

Comment:
is this export needed? it's not that useful publicly, just an internal shorthand so the `memoize` declaration can stay cleaner

Reply:
Sorted